cli/helm/chart.go

```diff
@@ -86,11 +93,7 @@ func readFile(chart embed.FS, f string, pathPrefix string) (*loader.BufferedFile
 	if err != nil {
 		return nil, err
 	}
-	// Remove the path prefix.
```

Comment:
`Rel()` doesn't exist in the `path` package. I looked at copying it from `filepath` but it uses a lot of internal stuff to `filepath` so not an easy copy. I think we're okay just trimming the prefix here because we're always passing in pathPrefix (hence variable name), not something we need to properly calculate the relative directory of.